postgres/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+name: harmony-postgres
+services:
+  postgres:
+    image: postgres:14
+    restart: always
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER:-postgres}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB:-postgres}
+    ports:
+      - "5432:5432"
+    volumes:
+      - ./postgresql/data:/var/lib/postgresql/data
```

Comment:
@Sybrand similar to prior comments, I left these exactly the same as per the setup on the zenysis codebase.
I completely agree with your reasoning. I just want to keep these changes tracked (or separate), if we make it here lets apply on internal.

For the bind mount, do use [this](https://www.postgresql.org/docs/9.1/creating-cluster.html) as reference for creating a host dir.

Reply:
@Ideonvn that was my initial thought on Friday - but then I started wondering - is `/usr/local/pgsql/data` ever going to cause a clash with someone running postgres locally / or other instances of dockerized postgres running on the same machine.

Would it not be safer to keep it separate?

Reply:
I won't be too worried about a local postgres clash - the bigger problem with having it on a relative dir is someone accidentally just deleting it.

However if you want to avoid it all, you can also use the convention of `/usr/local/pgsql_harmony/data`.